stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if or (eq .Values.controller.kind "Deployment") (eq .Values.controller.kind "Both") }}
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
Same as previous comment.

> I think this could potentially be a breaking change for some users, so can we just make this configurable with the default being the existing API version?

Reply:
The **Deployment** resource is no longer available in `v1.16` under the `extensions/v1beta1 ` API. It is now only available in the `apps/v1` API. This was available in the `apps/v1` API since `v1.9`. This should be backward compatible as long as `.spec.selector` is explicitly defined within the **Deployment** resource.

Source: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/